planner_cspace/src/planner_3d.cpp

```diff
@@ -1715,16 +1732,26 @@ class Planner3dNode
         {
           if (it != it_prev)
           {
-            const float d =
-                CyclicVecFloat<3, 2>(p).distLinestrip2d(*it_prev, *it);
-            if (d < d_min)
-              d_min = d;
+            int yaw = lroundf((*it)[2]) % map_info_.angle;
```

Comment:
is this thread-safe?

Reply:
Yes, multi-threading is used only in path search and distance map update function which blocks main thread.
Any other functions are run in single-thread.